modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1027,17 +997,29 @@ private CompletableFuture<Void> startPartitionAndStartClient(
             startGroupFut = falseCompletedFuture();
         }
 
-        startGroupFut
+        return startGroupFut
                 // TODO: the stage will be removed after https://issues.apache.org/jira/browse/IGNITE-22315
                 .thenComposeAsync(isReplicaStarted -> inBusyLock(busyLock, () -> {
                     if (isReplicaStarted) {
                         return nullCompletedFuture();
                     }
 
+                    // TODO: will be removed in https://issues.apache.org/jira/browse/IGNITE-22315
+                    Supplier<RaftGroupService> getCachedRaftClient = () -> {
```

Comment:
It's not cached.

Reply:
This is my temporal lambda that is about to remove

Reply:
This variable's name was here before my changes, do you want me to rename it?

Reply:
Nope.